prov/efa/src/rxr/rxr_op_entry.h

```diff
@@ -166,7 +168,7 @@ struct rxr_op_entry {
 	struct dlist_entry pending_recv_entry;
 #endif
 
-	size_t efa_outstanding_tx_ops; 
```

Comment:
This format change is nice, but should not be in this commit

Reply:
done